GoMeet/src/main/webapp/js/meeting-location-dao.js

```diff
@@ -21,7 +22,14 @@ class MeetingLocationDAO {
     params.append('lat', lat);
     params.append('lng', lng);
     params.append('note', note);
-    let result = await fetch(this.endpoint, {method: 'POST', body: params}).then((response) => response.json());
+    let result = await fetch(this.storingEndPoint, {method: 'POST', body: params}).then((response) => response.json());
     return result;
   }
+
+  /** Sends a request to the servlet to update a location entity's voteCount.*/
+  static async updateLocation(keyString) {
```

Comment:
Similar to my [comment](https://github.com/googleinterns/step271-2020/pull/28#discussion_r567499771) in PR28, it would be clear to name this function more specifically (e.g. voteLocation(keyString)), since it is only intended to increments the voteCount.